providers/gitlab/resources/gitlab.lr

```diff
@@ -82,4 +82,12 @@ gitlab.project @defaults("fullName visibility webURL") {
   autoDevopsEnabled bool
 	// Whether the requirements feature is enabled
   requirementsEnabled bool
+  // Merge methods for the project
+  mergeMethod() gitlab.project.mergeMethod
 }
+
+// GitLab project merge method
+gitlab.project.mergeMethod @defaults("method") {
```

Comment:
are you expecting to have more fields in this resource in the future? 
if not, `mergeMethod()` could just become a string

Reply:
You are right, it is better keet it simple. done. Thanks